groups/unifiedmanager/ec2.tf

```diff
@@ -26,6 +26,20 @@ module "unified_manager_ec2_security_group" {
       protocol    = "udp"
       description = "Syslog Collector"
       cidr_blocks = join(",", local.admin_cidrs)
+    },
+    {
+      from_port   = 443
+      to_port     = 443
+      protocol    = "tcp"
+      description = "iBoss access"
+      cidr_blocks = join(",", local.iboss_cidr)
+    },
+    {
+      from_port   = 22
+      to_port     = 22
+      protocol    = "tcp"
+      description = "iBoss access"
+      cidr_blocks = join(",", local.iboss_cidr)
```

Comment:
Rather than separate inline rules, this could be incorporated in to the `local.admin_cidrs` as defined in line 13. Otherwise we're unnecessarily duplicating work.

That being said, the `local.admin_cidrs` data is based on Vault data that can now be wholly replaced with the `administration-cidr-ranges` Managed Prefix List. This prefix list will contain all the necessary CIDRs required and can be specified in place of `ingress_cidr_blocks` on line 13 ingress_prefix_list_ids` [module argument](https://registry.terraform.io/modules/terraform-aws-modules/security-group/aws/3.0.0).